data/oecd/oecd_tariffs.ipynb

```diff
@@ -110,8 +110,110 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Remove spaces from column names\n",
-    "df.rename({\"Small Hydro\": \"Small_Hydro\", \"Solar PV\": \"Solar_PV\"}, axis=1, inplace=True)"
+    "# Rename year and country. Remove spaces from column names\n",
+    "df.rename({\"Year\": \"year\", \"Country\": \"country\", \"Small Hydro\": \"Small_Hydro\", \"Solar PV\": \"Solar_PV\"}, axis=1, inplace=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "612491d9-a6b0-4f09-a679-0e00cab83ab1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5a27e0a2-b4fb-4b49-9ce1-fb21234fdf97",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pwd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e787a3e9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load country mapping to ISO code\n",
+    "metadata_dir = \"metadata\"\n",
+    "path_to_country_conversion_table = os.path.join(metadata_dir, \"country_name_conversion.csv\")\n",
+    "if not os.path.exists(path_to_country_conversion_table):\n",
+    "  # get world bank country names and iso alpha3 code\n",
+    "  countries_table = pd.read_csv('../worldbank/processed/ccdr.csv')\n",
```

Comment:
I had to update the path to `ccdr.csv` here. Ideally the OECD notebook should not depend on the World Bank one, but we can take care of that later. By using an independent file, e.g. https://github.com/lukes/ISO-3166-Countries-with-Regional-Codes/blob/master/all/all.csv